### PR TITLE
Inbound transfer api integration

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- inbound transfer api integration
- publishing zeebe variables

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@avikganguly01 @fynmanoj

File: src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java [hunk 1]
```diff
@@ -635,32 +635,28 @@ private void acknowledgementRoutes(){
...
-                    {
-                        amsURL=paygopsHost;
-                    }
-                    else if(finalAmsVal.equalsIgnoreCase("roster"))
-                    {
-                        amsURL=rosterHost;
-                    }
-                    logger.debug("Final Value for ams : " + finalAmsVal);
-                    logger.debug("AMS URL : {}",amsURL);
+                    String amsURL = e.getIn().getHeader("amsUrl").toString();
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java:645]: This URL header should be removed.

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by fynmanoj** [comment]
Please rebase

File: src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java (hunk 1 above)
Comment (truthfool) [reply src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java:645]: This helps to call the respective ams which is passed from mpesa to channel.

**Pull request merged**